builds/misc/images.yaml

```diff
@@ -155,9 +155,9 @@ jobs:
      # Metrics Collector
       - template: templates/image-linux.yaml
         parameters:
-          name: Metrics Collector
+          name: Metrics Collector - Tests only
           imageName: azureiotedge-test-metrics-collectorr
-          project: MetricsCollector
```

Comment:
This project param needs to stay as MetricsCollector because that is how the Test Metrics Collector is referenced in the main csproj which defines all C# projects in our monorepo.

Reply:
Return to the original name.